### PR TITLE
Add support for skipping a crate

### PR DESCRIPTION
**Pull request by Manishearth**
Large projects may have a couple subcrates that are published on independent schedules, and the git log for those crates will end up showing those publishes as "changes". It's useful to be able to skip these.

### CODE REVIEW

**Comment by pksunkara** [comment]
I don't mind merging this but I am wondering if https://github.com/pksunkara/cargo-workspaces#fixed-or-independent doesn't solve the problem.

File: cargo-workspaces/src/utils/version.rs [hunk 1]
```diff
@@ -251,11 +257,13 @@ impl VersionOpt {
...
     }
 
-    fn ask_version(&self, cur_version: &Version, pkg_name: Option<&str>) -> Result<Version> {
+    /// Returns Ok(None) for skip option
+    fn ask_version(&self, cur_version: &Version, pkg_name: Option<&str>) -> Result<Option<Version>> {
         let mut items = version_items(cur_version, &self.pre_id);
 
         items.push(("Custom Prerelease".to_string(), None));
         items.push(("Custom Version".to_string(), None));
+        items.push((format!("Skip (stays {cur_version})"), None));
```
Comment (pksunkara) [inline cargo-workspaces/src/utils/version.rs:266]: Should be above "custom prerelease"

**Comment by Manishearth** [comment]
It does not: We already use that.

The problem is that cargo-workspaces doesn't know when these independent crates were last updated; it just does a diff against the given Git SHA and notices that there have been changes.


(Nor do I think that cargo-workspaces should try and figure this out, it really depends on the precise crate and users, having this option is useful)

**Comment by Manishearth** [comment]
Thanks! We tried using this with the latest ICU4X release and it worked pretty nicely.

**Comment by pksunkara** [comment]
Release 0.2.37.  Thanks for the PR!

Please let me know if you guys need anything else.

**Pull request merged**